### PR TITLE
feat: add kustomize in deployment

### PR DESCRIPTION
**Pull request by guilhem**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

This allows deployment with a kustomize process

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  

**Special notes for your reviewer**:

Only add in the latest version (v1.0.0) and master.
Should it be more?

**Release note**:
```
feat: add kustomize in deployment
```

### CODE REVIEW

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: deploy/v1.0.0/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (andyzhangx) [inline deploy/v1.0.0/kustomization.yaml:1]: pls only make change to `deploy` folder, `v1.0.0` is already released, should not change.

Comment (guilhem) [reply deploy/v1.0.0/kustomization.yaml]: @andyzhangx done :)

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve
thanks.

**Pull request merged**